tests/test_failure_propagator.py

```diff
@@ -52,27 +52,46 @@ def test_report_skip_triggering_test(self, request):
         """
         number_of_eligible_tests = config.RUN.get("number_of_tests") - 2
 
-        skipped_on_ceph_health_percent = (
-            config.RUN.get("skipped_tests_ceph_health") / number_of_eligible_tests
-        )
-        message = (
-            f"This run had {skipped_on_ceph_health_percent * 100}% of the "
-            f"tests skipped due to Ceph health not OK."
-        )
-        if skipped_on_ceph_health_percent > 0.25:
-            if config.RUN.get("skip_reason_test_found"):
-                test_name = config.RUN.get("skip_reason_test_found").get("test_name")
-                message = (
-                    message + f" The test that is likely to cause this is {test_name}"
-                )
-                squad = config.RUN.get("skip_reason_test_found").get("squad")
-                if squad:
-                    message = message + f" which is under {squad}'s responsibility"
-                    request.node.add_marker(squad)
+        # for acceptance suite, the value of config.RUN["skipped_on_ceph_health_threshold"] would be set to 0
+        # so any skip on Ceph health during an acceptance suite execution would cause
+        # test_report_skip_triggering_test to fail
+        if config.RUN.get("cli_params").get("-m") == "acceptance":
```

Comment:
Here it will ignore the case, if combination of `acceptance` and some other testsuite will be triggered (e.g. `acceptance or tier1`) - but I think it is not much likely production job configuration so it is ok to leave it like this.

Reply:
```suggestion
        if "acceptance" in config.RUN.get("cli_params").get("-m", "") :
```
Should resolve it.